### PR TITLE
Added missing DOIs and URLs

### PR DESCRIPTION
**Pull request by guaje**
This PR addresses the comments of Pham Van Vung and adds some missing DOIs and URLs.

https://github.com/openjournals/joss-reviews/issues/3384#issuecomment-871578718

### CODE REVIEW

File: paper.bib [hunk 1]
```diff
@@ -133,81 +125,40 @@ @article{ramachandran2011mayavi
...
-    month = nov,
-    volume = 595,
-    doi = {10.1051/0004-6361/201629272},
-    url = {http://adsabs.harvard.edu/abs/2016A%26A...595A...1G},
+@book{schroeder2004visualization,
+  title={The visualization toolkit: an object-oriented approach to 3D graphics},
+  author={Schroeder, Will J and Lorensen, Bill and Martin, Ken},
+  year={2004},
+  publisher={Kitware}
 }
```
Comment (skoudoro) [inline paper.bib:140]: can you replace by:
```tex
@inproceedings{schroeder1996visualization,
  author={Schroeder, W.J. and Martin, K.M. and Lorensen, W.E.},
  booktitle={Proceedings of Seventh Annual IEEE Visualization '96},
  title={The design and implementation of an object-oriented toolkit for 3D graphics and visualization},
  year={1996},
  volume={},
  number={},
  pages={93-100},
  doi={10.1109/VISUAL.1996.567752}
}
```

File: paper.bib [hunk 2]
```diff
@@ -133,81 +125,40 @@ @article{ramachandran2011mayavi
...
-  url = {https://github.com/arfon/fidgit}
+@article{virtanen2020scipy,
+  title={SciPy 1.0: fundamental algorithms for scientific computing in Python},
+  author={Virtanen, Pauli and Gommers, Ralf and Oliphant, Travis E and Haberland, Matt and Reddy, Tyler and Cournapeau, David and Burovski, Evgeni and Peterson, Pearu and Weckesser, Warren and Bright, Jonathan and others},
+  journal={Nature methods},
+  volume={17},
+  number={3},
+  pages={261--272},
+  year={2020},
+  publisher={Nature Publishing Group}
```
Comment (skoudoro) [inline paper.bib:157]: ```doi={10.1038/s41592-019-0686-2}```

File: paper.bib [hunk 3]
```diff
@@ -1,58 +1,38 @@
...
+@article{coumans2013bullet,
+  author  = {Coumans, Erwin and others},
+  journal = {Open source: bulletphysics. org},
+  title   = {Bullet physics library},
+  year    = {2013},
+  number  = {49},
+  pages   = {5},
+  volume  = {15},
+  url     = {http://www.bulletphysics.org},
 }
```
Comment (skoudoro) [inline paper.bib:25]: not sure if the spaces are ok or will generate an error

**Review by skoudoro — COMMENTED** [review]
Hi @guaje,

See below for some additional DOI.  It would be great if we could find the one for glumpy and scikit-learn

File: paper.bib (hunk 2 above)
Comment (guaje) [reply paper.bib:157]: Added.

File: paper.bib (hunk 3 above)
Comment (guaje) [reply paper.bib]: Those spaces were added by a bibtex manager but I removed them for consistency purposes.

File: paper.bib (hunk 1 above)
Comment (guaje) [reply paper.bib:140]: This is a new reference that cites the 1996 paper.

Comment (skoudoro) [reply paper.bib:140]: This is the same but older (the older one is the paper, and the other one is the book if I am not wrong). I personally prefer this reference.

Comment (guaje) [reply paper.bib]: Added.

**Review by skoudoro — APPROVED** [review]
LGTM! I think we should merge this and rerender the paper on JOSS

**Comment by guaje** [comment]
> LGTM! I think we should merge this and rerender the paper on JOSS

I've added DOIs and URLs to most of the references. However, I could not find a good way to add indexed references for frameworks or APIs such as Vulkan or PhysX.

**Comment by Garyfallidis** [comment]
Thank you @guaje !

**Pull request merged**